plugins/slicer/MONAILabel/MONAILabelLib/client.py

```diff
@@ -342,19 +341,20 @@ def send_response(conn, content_type="application/json"):
         logging.debug(f"HTTP Response Headers: {response.getheaders()}")
 
         response_content_type = response.getheader("content-type", content_type)
+        content_disposition = response.getheader("content-disposition")
```

Comment:
Return response headers.. that way users can use what they want

Reply:
Yup, response.headers is returned and users can get key-values from the dict.